CONTRIBUTING.md

```diff
@@ -439,10 +439,15 @@ The API reference is manually assembled in `doc/api/index.rst`.
 The *autodoc* sphinx extension will automatically create pages for each
 function/class/module listed there.
 
-You can reference classes, functions, and modules from anywhere (including docstrings)
-using <code>:func:\`package.module.function\`</code>,
-<code>:class:\`package.module.class\`</code>, or
+You can reference classes, functions, methods, and modules from anywhere 
+(including docstrings) using <code>:func:\`package.module.function\`</code>,
+<code>:class:\`package.module.class\`</code>, 
+<code>:meth:\`package.module.method\`</code>, or
 <code>:mod:\`package.module\`</code>.
+
+GMT documentation can be linked with 
+<code>:gmt-docs:\`page_name.html\`</code>. An example would be using
+<code>\`:gmt-docs:\`makecpt.html\`\`</code> to link to :gmt-docs:`makecpt.html`.
```

Comment:
There's also `:gmt-term:` linking to gmt.conf. See e.g. https://github.com/GenericMappingTools/pygmt/blob/2aa4aab1cc5cf925bbe01c9114d79ea9e1cb1a96/pygmt/mathops.py#L50